batch-nft-reveal/app/README.md

```diff
@@ -0,0 +1,48 @@
+# Batch NFT Reveal App
+
+[LIVE DEMO](#)
+
+This is a companion app for the [Chainlink Keepers Template: Batch NFT Reveal](https://github.com/hackbg/chainlink-keepers-templates/tree/main/batch-nft-reveal) contract repo. It helps you setup and play with batch-revealed NFT collections powered by Chainlink VRF & Keepers.
```

Comment:
Is this term "companion" a standard? I don't think it fits in our context. It could just be me. If the other reviewers like it I'm fine with that. It just looked weird when I first time read it.

Reply:
Suggestions?

Reply:
Аssociate / Additional ? Companion works too if you think it's better than these suggestions.